js/pricing.js

```diff
@@ -28,6 +58,67 @@ $(
                 disabledElement.attr('disabled', true);
                 disabledElement.addClass('disabled');
             }
+        };
+
+        /**
+         * Submit order handler.
+         */
+        const submitOrder = () => {
+            const dataProcessingConsent = $confirmPersonalInformation.prop("checked");
+            const supportAgreementConsent = $confirmDevelopmentAgreement.prop("checked");
+            const data = JSON.stringify({
+                "dataProcessingConsent": dataProcessingConsent,
+                "supportAgreementConsent": supportAgreementConsent
+            });
+
+            const transactionUrl = apiUrl + registerTransactionPath;
+            sendPaymentTransaction (transactionUrl, data);
+        };
+
+        /**
+         * Sends the transaction data and returns the transaction ID. If the request is successful,
+         * redirects to the Payment screen.
+         *
+         * @param {String} transactionUrl - the API URL
+         * @param {Object} data - the transaction data
+         */
+        const sendPaymentTransaction = (transactionUrl, data) => {
+            showRedirect(false);
+            $.ajax(transactionUrl, {
+                type: 'POST',
+                data: data,
+                contentType: 'application/json',
+                success: (data) => {
+                    const obj = JSON.parse(data);
+                    const paymentUrl = orderUrl + '&CUSTOMERID=' + obj.id;
+                    window.location = paymentUrl;
+                    hideRedirect();
+                },
+                error: (jqXhr) => {
+                    let errorMessage = jqXhr.status + ': ' + jqXhr.statusText;
+                    console.debug(errorMessage);
+                    showRedirect(true);
+                }
+            });
+        };
+
+        /**
+         * Shows the redirect screen.
+         * @param {Boolean} isError - if it is true, hides the loader and shows the error section
```

Comment:
Please separate `@param` with an empty line from the function description itself.